### DISCUSSION TITLE
[ODM]  orderByDate

### DISCUSSION THREAD

**Discussion by wstrange** [firebase/flutterfire#8960] [Feature request] [answered]
The ODM generator generates orderBy for most string fields, but not for DateTime. 

Dates are stored as iso formatted strings - a string based orderBy should work?

**Comment by wstrange** [comment]
[Meant to put this in the ODM thread - apologies - But I cant delete this..]

**Comment by rrousselGit** [comment, marked as answer]
There's already a PR for DateTime support. It's just currently waiting for another feature to land first
